docs/ref/urls.txt

```diff
@@ -5,7 +5,21 @@
 .. module:: django.urls.conf
     :synopsis: Functions for use in URLconfs.
 
-.. currentmodule:: django.conf.urls
+.. currentmodule:: django.urls
+
+``path()``
+==========
+
+.. function:: path(regex, view, kwargs=None, name=None)
+
+...
+
+``re_path()``
+=============
+
+.. function:: re_path(regex, view, kwargs=None, name=None, converters=None)
```

Comment:
As noted in https://github.com/django/django/pull/7482#discussion_r138841581, the `converters` argument is currently unused. I'm unsure if it should remain or not.